app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    counter = Comment.where(post_id:).count
+    post = Post.find_by(id: post_id)
+    post.update(comments_counter: counter)
+  end
```

Comment:
- [ ]  A faster way (performance wise) would be:
```rb
def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
✅ Resolved.